Script/test-SPM.sh

```diff
@@ -67,14 +67,26 @@ xcodebuild archive -scheme TestProject -destination 'generic/platform=iOS'
 echo '############# Build for generic iOS device ###############'
 xcodebuild build -scheme TestProject -destination 'generic/platform=iOS'
 
-# Build for i386 simulator
-echo '############# Build for i386 simulator ###############'
+# Build for i386 iOS simulator
+echo '############# Build for i386 iOS simulator ###############'
 xcodebuild build -scheme TestProject -destination 'generic/platform=iOS Simulator' ARCHS=i386
```

Comment:
Architecture i386 is deprecated in Xcode 14, so this script will fail at this step. As this is a branch merge, we can fix this in another PR if needed.

Reply:
Yes, we will update this along with the version bump in the separate PR